lib/block-supports/typography.php

```diff
@@ -88,13 +88,11 @@ function gutenberg_apply_typography_support( $block_type, $block_attributes ) {
 	$has_text_decoration_support = _wp_array_get( $typography_supports, array( '__experimentalTextDecoration' ), false );
 	$has_text_transform_support  = _wp_array_get( $typography_supports, array( '__experimentalTextTransform' ), false );
 
-	$skip_font_size_support_serialization = _wp_array_get( $block_type->supports, array( '__experimentalSkipFontSizeSerialization' ), false );
-
-	// Covers all typography features _except_ font size.
-	$skip_typography_serialization = _wp_array_get( $block_type->supports, array( '__experimentalSkipTypographySerialization' ), false );
+	// Covers all typography features.
+	$skip_typography_serialization = _wp_array_get( $typography_supports, array( '__experimentalSkipSerialization' ), false );
```

Comment:
A couple of things here:

- the letter spacing does not use this flag but should (previous to this PR)
- instead of having this check for each individual feature, what if we check for `__experimentalSkipSerialization` after we do it for `$typography_supports`  and before each individual support and return early if there's nothing to do here?